iocore/net/OCSPStapling.cc

```diff
@@ -21,16 +21,10 @@
 
 #include "P_OCSPStapling.h"
 
-#if TS_USE_TLS_OCSP
-
-#include <openssl/bio.h>
 #include <openssl/ssl.h>
-
-#if TS_HAS_BORINGOCSP
-#include <boringocsp/ocsp.h>
-#else
-#include <openssl/ocsp.h>
-#endif
+#include <openssl/x509v3.h>
+#include <openssl/asn1.h>
+#include <openssl/asn1t.h>
```

Comment:
Is all this code copied from openssl 3.0?  If not, do we need to display this license somewhere?  https://www.openssl.org/source/license-openssl-ssleay.txt

Reply:
All code copied is from OpenSSL 3.0 or 3.1 (the both are Apache License), but we may still need some note about it. @zwoop What do you think?

Reply:
Added comments.